test/util/IdTableHelpers.h

```diff
@@ -99,12 +100,13 @@ have.
 generated row must ALWAYS have size `numberColumns`. Otherwise an exception will
 be thrown.
 */
-IdTable generateIdTable(const size_t numberRows, const size_t numberColumns,
-                        const std::function<IdTable::row_type()>& rowGenerator);
+IdTable generateIdTable(
+    const size_t numberRows, const size_t numberColumns,
+    const std::function<std::vector<ValueId>()>& rowGenerator);
 
 /*
-@brief Creates a `IdTable`, where the content of the join column is given via
-functions and all other columns are randomly filled with numbers.
+@brief Create a `IdTable`, where the content of the join columns are given via
```

Comment:
```suggestion
@brief Create an `IdTable`, where the content of the join columns are given via
```

Reply:
Fixed in [1256dd6](https://github.com/ad-freiburg/qlever/pull/1016/commits/1256dd6a35f1b18633e8ac0e80ac30695185ba3b).